src/components/HomeTabBtn/HomeTabBtn.jsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+
```

Comment:
Utwórz plik ButtonDelete.module.css w folderze ButtonDelete, w którym będziesz stylizować przycisk. 
Dodaj import pliku styli w tym pliku:
import css from './ButtonDelete.module.css';

W pliku ButtonDelete.module.css, importuj zmienne vars.css i tam dodaj stylizację, razem z kolorami, radiusami itp. ze zmiennych. Import w pliku ButtonDelette.module.css wyglądać będzie tak:
@import '../../stylesheet/vars.css';

tam dodasz klasę dla przycisku, wystarczy, jak to będzie:
.button {
stylizacja przycisku...
}

Reply:
komponent balance stylizowałam w ten sposób i Mari uznała go za poprawny, czy może zostać w ten sposób czy to poprawiać? co to zmiennym to będą zmienione